supplier_app/models.py

```diff
@@ -205,14 +210,20 @@ class TaxPayerArgentina(TaxPayer):
         upload_to='file',
         blank=False,
         verbose_name=_('AFIP registration certificate'),
-        validators=[FileExtensionValidator(allowed_extensions=['pdf'])],
```

Comment:
Maybe it is better to put the allowed_extensions equal to a variable so you can add other extensions easely in the future